lib/cmd/fhc/targets.js

```diff
@@ -130,26 +134,19 @@ function hasTarget(targets, target) {
 }
 
 function getTarget(target) {
-  if (!target.target.match(/\/$/)) target.target = target.target + '/';
-  var targ;
+  if (!target.target.match(/\/$/)) {
+    target.target = target.target + '/';
+  }
   var targets = getTargets();
 
-  for (var i=0; i<targets.length; i++) {
-    var t = targets[i];
-
-    if (t.target === target.target){
-      if (target.user !== undefined) {
-        if(target.user === t.user) {
-          targ = t;
-          break;
-        }
-      }else {
-        targ = t;
-        break;
+  return _.find(targets, function (t) {
+    if (target.target === t.target) {
+      if (target.user) {
+        return target.user === t.user;
```

Comment:
This suggestion is not really equivalent, since it doesn't check equality for `user` if it's falsey, but still returns true.